modules/mod_admin/templates/_admin_edit_content_seo.tpl

```diff
@@ -87,24 +87,54 @@
             </div>
         </div>
 
-        <div class="form-group row">
-            <div class="col-md-9 col-md-offset-3">
-                <div class="checkbox">
-                    <label>
-                        <input id="seo_noindex"
-                            type="checkbox"
-                            name="seo_noindex"
-                            value="1"
-                            {% if not is_editable %}disabled{% endif %}
-                            {% if id.seo_noindex %}checked="checked"{% endif %}
-                        />
-                        {_ Ask google to not index this page _}
-                    </label>
+        {% with id.category_id.is_seo_noindex_cat as is_seo_noindex_cat %}
+            <div class="form-group row">
+                <div class="col-md-9 col-md-offset-3">
+                    <div class="checkbox">
+                        <label>
+                            <input id="seo_noindex"
+                                type="checkbox"
+                                name="seo_noindex"
+                                value="1"
+                                {% if not is_editable %}disabled{% endif %}
+                                {% if id.seo_noindex or is_seo_noindex_cat %}checked="checked"{% endif %}
+                                {% if is_seo_noindex_cat %}disabled{% endif %}
```

Comment:
Should this be merged with the condition at line 99? The duplication probably doesn't matter that much actually, in practice.

Reply:
Ah good one, indeed better to merge.